src/components/Calculator.jsx

```diff
@@ -1,53 +1,128 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Button from './shared/Button/Button';
+import calculate from './logic/calculate';
 import './style.css';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+const Calculator = () => {
+  const [data, setData] = useState(0);
+  const onClick = (e) => {
+    setData((data) => calculate(data, e.target.value));
+  };
 
-  render() {
-    return (
-      <div className="wrapper flex align-center justify-center">
-        <div className="calculator">
-          <div className="answer-display">
-            <p>0</p>
-          </div>
-          <div className="row flex space-btn">
-            <Button className="button gray-bg flex-1" text="AC" />
-            <Button className="button gray-bg flex-1" text="+/-" />
-            <Button className="button gray-bg flex-1" text="%" />
-            <Button className="button orange-bg flex-1" text="/" />
```

Comment:
The calculator app interface disappears due to errors when the user tries to use the `/` button for division. Kindly see the console errors below for reference.
 
![div_error](https://user-images.githubusercontent.com/35504149/177801975-609516d8-6d0a-4fb9-9a27-2b8d298aad28.png)